### DISCUSSION TITLE
Expandable Section

### DISCUSSION THREAD

**Discussion by ux-fran** [nl-design-system/backlog#344] [Component suggestions]
![afbeelding](https://github.com/user-attachments/assets/bf1dbc77-254d-4b6d-95ba-058422a7a98d)

We willen graag meer leren over de Expandable Section component. In welke vorm, of vormen, wordt deze momenteel ingezet en toegepast binnen jouw organisatie?

We beschouwen deze component niet als Accordion, omdat de Accordion wellicht beter gebruikt kan worden voor beknopte content, en de Expandable section hele lijsten en formulieren zou kunnen bevatten.

[Figma - Overheidsbrede portalen](https://www.figma.com/design/iZgSIuU8hvH9nw3h7WO1ZY/Overheidsbrede-portalen---Templates?node-id=4479-106721)

Daarnaast wordt het delen van onderzoek gewaardeerd. Dus heb je zelf onderzoek gedaan en zijn er inzichten voor dit component? Dan horen we dit ook graag!

Zie ook de [Slack discussie](https://codefornl.slack.com/archives/C7KDQRXMZ/p1728300535723359)

**Comment by ux-fran** [comment]
## Research okt 2024
-  Gekeken in diverse files van Amsterdam, DSO, DUO, Open Formulieren, Open Gemeenten, Rotterdam, RVO en Utrecht en verder iets vergelijkbaars gevonden.

Heb ik wat gemist of weet jij (de lezer) misschien nog een organisatie of andere gemeente met een vergelijkbaar component, vul gerust aan!

**Comment by ux-fran** [comment]
## Den Haag

- [Figma ](https://www.figma.com/design/MeR4pOHknHd5ay7Oi0SrZs/%F0%9F%91%91-Mijnprofielservices---MijnDenHaag--Scope-hack-a-ton-2024?node-id=16207-31785&t=odpe2qIiQCexmKmZ-1)
---

### Screenshots / Varianten

**Mijn Gegevens**
![afbeelding](https://github.com/user-attachments/assets/a7c315a5-6939-46e8-860e-925c05b49e16)
![afbeelding](https://github.com/user-attachments/assets/51424331-4106-4826-959a-70d3b725f6ae)

![afbeelding](https://github.com/user-attachments/assets/604e8aed-6606-4009-bd87-1639a68d0de8)
![afbeelding](https://github.com/user-attachments/assets/29809ccd-8ec2-45e1-93e9-661b7b709a91)

![afbeelding](https://github.com/user-attachments/assets/5bd5ffe5-502b-422f-ae42-ffd9f66356f0)
![afbeelding](https://github.com/user-attachments/assets/1759e61b-0f8a-416a-8f8e-c3c7d3a76630)

![afbeelding](https://github.com/user-attachments/assets/68efe55e-a90d-4297-8d1b-0378ae92d1b6)
![afbeelding](https://github.com/user-attachments/assets/204af25a-ad59-4d25-ae3e-67d706b43640)

![afbeelding](https://github.com/user-attachments/assets/0522577f-2458-4395-bfa5-8ce5b409273b)
![afbeelding](https://github.com/user-attachments/assets/0aa2b5d4-8c19-4868-a442-881601d948e4)

![afbeelding](https://github.com/user-attachments/assets/1c568353-bf2f-40bd-a7f9-bda63ea6ed70)
![afbeelding](https://github.com/user-attachments/assets/49fed1a8-66ee-4cb6-a32c-5e28cf4d464a)

---

### Notities

-

**Comment by Rozerinay** [comment]
We sluiten de Discussion voor deze component, omdat deze vooralsnog wordt gezien als een snowflake. Er is weinig onderscheid tussen deze component en de [Accordion](https://github.com/orgs/nl-design-system/discussions/217). 

Mocht Expandable Section functionaliteiten bevatten die in de Accordion ontbreken, is het goed om dit aan te geven op het moment dat Accordion de status Candidate heeft. 

Goed om te weten is dat RVO al een implementatie heeft van de [Accordion met een 'teaser text'](https://github.com/orgs/nl-design-system/discussions/217#discussioncomment-8441072) die veel lijkt op de Expandable Section.
